<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.475">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>index</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

<script src="index_files/libs/core-js-2.5.3/shim.min.js"></script>
<script src="index_files/libs/react-17.0.0/react.min.js"></script>
<script src="index_files/libs/react-17.0.0/react-dom.min.js"></script>
<script src="index_files/libs/reactwidget-1.0.0/react-tools.js"></script>
<script src="index_files/libs/htmlwidgets-1.6.2/htmlwidgets.js"></script>
<link href="index_files/libs/reactable-0.4.4/reactable.css" rel="stylesheet">
<script src="index_files/libs/reactable-binding-0.4.4/reactable.js"></script>


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-full">

<main class="content column-page" id="quarto-document-content">



<section id="overview-table-of-dl-based-survival-methods" class="level1">
<h1>Overview Table of DL-based Survival Methods</h1>
<div class="cell">
<div class="cell-output-display">

<button onclick="Reactable.setHiddenColumns('table', prevColumns => { return prevColumns.length === 0 ? ['authors', 'year', 'multiple losses', 'type of interpretability', 'code availability', 'code repository', 'datasets (non-private)', 'simulation'] : [] })">Show/hide more columns</button>
<button onclick="Reactable.downloadDataCSV('table', 'table.csv', {sep: ';'})">
<svg aria-hidden="true" role="img" viewbox="0 0 512 512" style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:currentColor;overflow:visible;position:relative;"><path d="M288 32c0-17.7-14.3-32-32-32s-32 14.3-32 32V274.7l-73.4-73.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3l128 128c12.5 12.5 32.8 12.5 45.3 0l128-128c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L288 274.7V32zM64 352c-35.3 0-64 28.7-64 64v32c0 35.3 28.7 64 64 64H448c35.3 0 64-28.7 64-64V416c0-35.3-28.7-64-64-64H346.5l-45.3 45.3c-25 25-65.5 25-90.5 0L165.5 352H64zM432 456c-13.3 0-24-10.7-24-24s10.7-24 24-24s24 10.7 24 24s-10.7 24-24 24z"></path></svg>
Download as CSV
</button>
<div id="table" class="reactable html-widget " style="width:auto;height:700px;"></div>
<script type="application/json" data-for="table">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"title":["Individual Survival Curves with Conditional Normalizing Flows","CapSurv: Capsule Network for Survival Analysis With Whole Slide Pathological Images","A convolutional neural network model for survival prediction based on prognosis-related cascaded Wx feature selection","CNN-based survival model for pancreatic ductal adenocarcinoma in medical imaging","Survival Analysis with High-Dimensional Omics Data Using a Threshold Gradient Descent Regularization-Based Neural Network Approach","Deep learning based feature-level integration of multi-omics data for breast cancer patients survival analysis","Cox-nnet: An artificial neural network method for prognosis prediction of high-throughput omics data","Cox-PASNet: Pathway-based Sparse Deep Neural Network for Survival Analysis","Time-to-Event Prediction with Neural Networks and Cox Regression","CRESA: A Deep Learning Approach to Competing Risks, Recurrent Event Survival Analysis","DAFT: A universal module to interweave tabular data and 3D images in CNNs","A Deep Active Survival Analysis approach for precision treatment recommendations: Application of prostate cancer","Adversarial Time-to-Event Modeling","Deep Cox Mixtures for Survival Regression","Deep Learning-Based Discrete Calibrated Survival Prediction","DeepComp: Which Competing Event Will Hit the Patient First?","DeepCompete : A deep learning approach to competing risks in continuous time domain","Deep convolutional neural network for survival analysis with pathological images","DeepHit: A Deep Learning Approach to Survival Analysis with Competing Risks","DeepOmix: A scalable and interpretable multi-omics deep learning framework and application in cancer survival analysis","DeepPAMM: Deep Piecewise Exponential Additive Mixed Models for Complex Hazard Structures in Survival Analysis","DeepSurv: personalized treatment recommender system using a Cox proportional hazards deep neural network","Estimation of Conditional Mixture Weibull Distribution with Right Censored Data Using Neural Network for Time-to-Event Analysis","Genome-wide association study-based deep learning for survival prediction","DNNSurv: Deep Neural Networks for Survival Analysis Using Pseudo Values","DPWTE: A Deep Learning Approach to Survival Analysis Using a Parsimonious Mixture of Weibull Distributions","Deep Recurrent Survival Analysis","Deep Survival Machines: Fully Parametric Survival Regression and Representation Learning for Censored Data with Competing Risks","Extreme learning machine Cox model for high-dimensionalsurvival analysis","Group Lasso Regularized Deep Learning for Cancer Prognosis from Multi-Omics and Clinical Features","Image-Based Survival Prediction for Lung Cancer Patients Using CNNS","Transformer-Based Deep Survival Analysis","Deep Integrative Analysis for Survival Prediction","IDNetwork: A deep Illness-Death Network based on multi-state event history process for versatile disease prognostication","A Multi-constraint Deep Semi-supervised Learning Method for Ovarian Cancer Prognosis Prediction","MultiSurv: Long-term cancer survival prediction using multimodal deep learning","Deep Neural Networks for Survival Analysis Based on a Multi-Task Framework","Deep survival algorithm based on nuclear norm","A scalable discrete-time survival model for neural networks","Continuous and Discrete-Time Survival Prediction with Neural Networks","A meta-learning approach for genomic survival analysis","A deep survival analysis method based on ranking","Survival prediction model for right-censored data based on improved composite quantile regression neural network","RNN-SURV: A Deep Recurrent Model for Survival Analysis","SALMON: Survival Analysis Learning With Multi-Omics Neural Networks on Breast Cancer","Survival Analysis For Computing Systems Using A Deep Ensemble Network","Survival Prediction Based on Histopathology Imaging and Clinical Data: A Novel, Whole Slide CNN Approach","Deep Semisupervised Multitask Learning Model and Its Interpretability for Survival Analysis","DeepBTS: Prediction of Recurrence-free Survival of Non-small Cell Lung Cancer Using a Time-binned Deep Neural Network","SurvCNN: A Discrete Time-to-Event Cancer Survival Estimation Framework Using Image Representations of Omics Data","SurvNet: A Novel Deep Neural Network for Lung Cancer Survival Analysis With Missing Values","A General Framework for Survival Analysis and Multi-State Modelling","SurvTRACE: Transformers for Survival Analysis with Competing Events","Discrete-time survival analysis in the critically ill: a deep learning approach using heterogeneous data","Deep learning for the dynamic prediction of multivariate longitudinal and survival data","Improved survival analysis by learning shared genomic information from pan-cancer data","A Wide and Deep Neural Network for Survival Analysis from Anatomical Shape and Tabular Clinical Data","Mixture cure rate models with neural network estimated nonparametric components"],"authors":["G. Ausset et al.","B. Tang et al.","Q. Yin et al.","Y. Zhang et al.","Y. Fanet al.","L. Tong et al.","T. Ching et al.","J. Hao et al.","H. Kvamme et al.","G. Gupta et al.","T. Wolf et al.","M. Nezhad et al.","P. Chapfuwa et al.","C. Nagpal et al.","P. Fuhlert et al.","Y. Li et al.","Aastha et al.","X. Zhu et al.","C. Lee et al.","L. Zhao et al.","P. Kopper et al.","J. Katzman et al.","A. Bennis et al.","T. Sun","L. Zhao et al.","A. Bennis et al.","K. Ren et al.","C. Nagpal et al.","H. Wang et al.","G. Xie et al.","C. Haarburger et al.","S. Hu et al.","C. Huang et al.","A. Cottin et al.","H. Chai et al.","L. Vale-Silva and K. Rohr","S. Fotso","J. Tong and X. Zhao","M. Gensheimer and B. Narasimhan","H. Kvamme and Ø. Borgan ","Y. Qiu et al.","B. Jing et al.","X. Qin et al.","E. Giunchiglia et al.","Z. Huang et al.","B. Sansaengtham et al.","S. Agarwal et al.","S. Chi et al.","B. Lee et al.","Y. Kalakoti et al.","J. Wang et al.","S. Groha et al.","Z. Wang","H. Thorsen-Meyer et al.","J. Lin and S. Luo","S. Kim et al.","S. Pölsterl et al.","Y. Xie and Z. Yu"],"year":[2021,2019,2022,2020,2022,2020,2018,2018,2019,2019,2022,2019,2018,2021,2022,2020,2021,2016,2018,2021,2022,2018,2017,2020,2019,2021,2019,2021,2019,2019,2019,2021,2018,2022,2022,2021,2018,2022,2019,2021,2020,2019,2022,2018,2019,2020,2021,2021,2020,2021,2021,2021,2021,2022,2022,2020,2020,2021],"model name":["Aus2021","CapSurv","CNN-Cox","CNN-Survival","CNT","ConcatAE / CrossAE","Cox-nnet","Cox-PASNet","Cox-Time","CRESA","DAFT","DASA","DATE","DCM","DCS","DeepComp","DeepCompete","DeepConvSurv","DeepHit","DeepOmix","DeepPAMM","DeepSurv","DeepWeiSurv","DNNSurv_Sun2022","DNNSurv_Zha2019","DPWTE","DRSA","DSM","ELMCoxBAR","GDP","Haa2019","Hu2021","Hua2018","IDNetwork","MCAP","MultiSurv","N-MTLR","NN-DeepSurv","Nnet-survival","PC-Hazard","Qiu2020","RankDeepSurv","rcICQRNN","RNN-SURV","SALMON","San2020","SSCNN","SSMTL","su-DeepBTS","SurvCNN","SurvNet","survNode","SurvTRACE","Tho2022","TransformerJM","VAECox","WideAndDeep","Xie2021"],"model class":["other","Cox-based","Cox-based","discrete-time","Cox-based","Cox-based","Cox-based","Cox-based","Cox-based","discrete-time","Cox-based","other","other","Cox-based","discrete-time","discrete-time","other","Cox-based","discrete-time","Cox-based","PEM-based","Cox-based","parametric","Cox-based","discrete-time","parametric","discrete-time","parametric","Cox-based","Cox-based","Cox-based","discrete-time","other","PEM-based","Cox-based","discrete-time","discrete-time","Cox-based","discrete-time","PEM-based","Cox-based","other","other","discrete-time","Cox-based","Cox-based","other","discrete-time","discrete-time","discrete-time","Cox-based","other","discrete-time","discrete-time","discrete-time","Cox-based","Cox-based","Cox-based"],"paramet-rization":["parameters of transformation model","hazard rate","hazard rate","discrete hazard","hazard rate","hazard rate","hazard rate","hazard rate","hazard rate","discrete hazard","hazard rate","n.a.","custom loss","hazard rate","discrete hazard","discrete hazard","hazard rate","hazard rate","PMF","hazard rate","hazard rate","hazard rate","(mixture) Weibull Loss","hazard rate","","(mixture) Weibull Loss","discrete hazard","(mixture) Weibull or Log-Normal loss","hazard rate","hazard rate","hazard rate","discrete hazard","person-specific risk","","hazard rate","discrete hazard","parameters of logistic regression","hazard rate","discrete hazard","hazard rate","hazard rate","custom loss","parameters of quantile regression","survival probability","hazard rate","hazard rate","custom loss function","custom loss function","discrete hazard","discrete hazard","hazard rate","hazard rate","discrete hazard","???","PMF","hazard rate","hazard rate","hazard rate"],"archi-tecture":["other","other","CNN","CNN","only FFNN","AE/VAE","only FFNN","other","only FFNN","RNN","CNN","framework","other","AE/VAE","RNN","RNN","nODE","CNN","RNN","only FFNN","flexible","only FFNN","only FFNN","only FFNN","only FFNN","other","RNN","RNN","other","only FFNN","CNN","transformer","CNN","only FFNN","only FFNN","CNN","only FFNN","only FFNN","flexible","only FFNN","only FFNN","other","only FFNN","RNN","other","other","CNN","only FFNN","only FFNN","CNN","other","nODE","transformer","RNN","transformer","AE/VAE","other","CNN"],"multiple losses":["yes","yes","no","no","no","yes","no","no","no","yes","no","n.a.","yes","yes","yes","yes","no","no","yes","no","no","no","no","no","no","no","yes","yes","no","no","no","yes","no","no","yes","no","not described","no","no","no","no","yes","no","yes","no","no","yes","yes","no","no","yes","no   ","yes","no","yes","yes","no","no"],"left-truncation":["no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no  ","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no"],"interval-censoring":["no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no   ","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no"],"CR":["no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","yes","yes","no","yes","no","yes","no","no","no","no","no","no","yes","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","yes","no","no","no","no","no","no"],"MSM":["no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no"],"recurrent events":["no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no"],"TVF":["no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","yes","no","yes","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","yes","no","no","no"],"TVE":["no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","yes","no","no","yes","no","no","no","no","yes","no","no","no","yes","no","no","no","no","no","no","no","no","no","no"],"high dimen-sionality":["no","no","yes","yes","yes","yes","yes","yes","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","yes","no","no","no","no","no","yes","yes","no","no","no","yes","yes","no","no","no","no","yes","no","yes","no","yes","no","no","no","no","yes","no","no","no","no","no","yes","no","no"],"multi-modality":["no","yes","no","yes","no","yes","no","no","no","no","yes","no","no","no","no","no","no","yes","no","no","yes","no","no","no","no","no","no","no","no","no","yes","no","yes","no","no","yes","no","no","yes","no","no","no","no","no","no","no","yes","no","no","no","no","yes","no","yes","no","no","yes","yes"],"interpret-ability":["no","no","no","no","no","no","assigning biological meaning to nodes","assigning biological meaning to nodes","no","no","no","no","no","no","no","no","no","no","no","assigning biological meaning to nodes","classical statistical interpretability of tabular data","no","no","LIME","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","risk backpropagation based on SurvivalNet","no","no","no","feature importance","no","no","feature importance; feature effect plots on cumulative incidence curves","no","no","no","variational survNode","interpretation of attention maps","SHAP","no","no","classical statistical interpretability of tabular data","classical statistical interpretability of tabular data"],"code":["https://git.sr.ht/~aussetg/nfsurvival","no","https://github.com/wangwangCCChen/CNN-Cox","no","https://github.com/shuanggema/CNTsurv","https://github.com/tongli1210/BreastCancerSurvivalIntegration","https://github.com/lanagarmire/cox-nnet","https://github.com/DataX-JieHao/Cox-PASNet","https://github.com/havakv/pycox, https://github.com/RaphaelS1/survivalmodels","no","https://github.com/ai-med/DAFT","no","https://github.com/paidamoyo/adversarial_time_to_event","https://github.com/chiragnagpal/deep_cox_mixtures, https://autonlab.org/auton-survival","https://github.com/imsb-uke/dcsurv","no","no","no","https://github.com/chl8856/DeepHit, https://github.com/havakv/pycox, https://github.com/RaphaelS1/survivalmodels\n","https://github.com/CancerProfiling/DeepOmix","no","https://github.com/jaredleekatzman/DeepSurv, https://github.com/havakv/pycox, https://github.com/RaphaelS1/survivalmodels, https://autonlab.org/auton-survival, https://square.github.io/pysurvival","no","https://github.com/yingding99/DNNSurv","https://github.com/lilizhaoUM/DNNSurv, https://github.com/RaphaelS1/survivalmodels","no","https://github.com/rk2900/drsa","https://autonlab.org/auton-survival","https://github.com/whcsu/ELMCoxBAR","https://github.com/WGLab/GDP","no","no","no","no","no","https://github.com/luisvalesilva/multisurv","https://square.github.io/pysurvival","no","https://github.com/MGensheimer/nnet-survival, https://github.com/havakv/pycox, https://github.com/RaphaelS1/survivalmodels","https://github.com/havakv/pycox, https://github.com/RaphaelS1/survivalmodels","https://github.com/gevaertlab/metalearning_survival","https://github.com/sysucc-ailab/RankDeepSurv","no","no","https://github.com/huangzhii/SALMON","no","no","https://github.com/LeyiChi/SSMTL","no","https://github.com/TeamSundar/SurvCNN","no","https://github.com/stefangroha/SurvNODE","https://github.com/RyanWangZf/SurvTRACE","https://github.com/daplaci/ICUsurvive","no","https://github.com/SunkyuKim/VAECox","no","no"],"datasets (non-private)":["METABRIC, RGBSG, SUPPORT, WHAS","TCGA","TCGA","Lung1 (TCIA)","HGSOC, TCGA","TCGA","TCGA","TCGA","FLCHAIN, KKBox churn, METABRIC, RGBSG, SUPPORT","MIMIC III","ADNI, AIBL","SEER","EHR, FLCHAIN, SEER, SUPPORT","FLCHAIN, SEER, SUPPORT","FLCHAIN, METABRIC, SUPPORT","CAFR, SEER","AML, DBCD, DLBCL, NSBCD, SEER","NLST","METABRIC, SEER, UNOS","TCGA","BREAST, GBSG2, ICU, METABRIC, MGUS2, TUMOR","METABRIC, RGBSG, SUPPORT, WHAS","METABRIC, SEER","AREDS","CHS, MESA","METABRIC, SEER, SUPPORT","BIDDING, CLINIC, MUSIC","METABRIC, SEER, SUPPORT","DrAsGiven, EMTAB386, GSE14764, GSE32062, GSE49997, LungBeer, METABRIC, NSBCD, TCGAmirna","TCGA","Lung1 (TCIA)","METABRIC, SUPPORT","TCGA","METABRIC, NCT00079274, NCT00275210","GSE17260, GSE26193, GSE26712, GSE32062, GSE53963, GSE63885, TCGA-OV","TCGA","WHAS, veteran","METABRIC, MIMIC III, MLC, RGBSG, SUPPORT, WHAS","SUPPORT","FLCHAIN, METABRIC, NWTCO, RGBSG, SUPPORT","TCGA","METABRIC, RGBSG, SUPPORT, WHAS","METABRIC, NKI70, TCC","AIDS2, FLCHAIN, NWTCO, UNOS Transplant, UNOS Waitlist","BRCA (GDAC Firehose)","Google Cloud Cluster Dataset","TCGA","SEER","METABRIC, GBSG, NCCTG Lung Cancer Data","TCGA","SEER","METABRIC, ESBMT Data","METABRIC, SUPPORT","Danish Electronic Patient Records (Registered Access)","ADNI, NACC","TCGA","ADNI","OASIS-3"],"simulation":["yes","no","no","no","yes","no","yes","no","yes","no","no","no","no","no","yes","no","no","no","yes","no","yes","yes","no","yes","yes","no","no","no","yes","yes","no","no","no","yes","no","no","yes","yes","yes","yes","no","no","yes","no","no","no","no","no","no","no","no","yes","no","no","yes","no","no","yes"]},"columns":[{"id":".details","name":"","type":null,"sortable":false,"resizable":false,"filterable":false,"searchable":false,"width":45,"align":"center","details":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null]},{"id":"title","name":"title","type":"character","show":true,"cell":[{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/9564222","target":"_blank"},"children":["Individual Survival Curves with Conditional Normalizing Flows"]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/8651474","target":"_blank"},"children":["CapSurv: Capsule Network for Survival Analysis With Whole Slide Pathological Images"]},{"name":"a","attribs":{"href":"https://www.nature.com/articles/s41374-022-00801-y","target":"_blank"},"children":["A convolutional neural network model for survival prediction based on prognosis-related cascaded Wx feature selection"]},{"name":"a","attribs":{"href":"https://bmcmedimaging.biomedcentral.com/articles/10.1186/s12880-020-0418-1","target":"_blank"},"children":["CNN-based survival model for pancreatic ductal adenocarcinoma in medical imaging"]},{"name":"a","attribs":{"href":"https://www.mdpi.com/2073-4425/13/9/1674","target":"_blank"},"children":["Survival Analysis with High-Dimensional Omics Data Using a Threshold Gradient Descent Regularization-Based Neural Network Approach"]},{"name":"a","attribs":{"href":"https://bmcmedinformdecismak.biomedcentral.com/articles/10.1186/s12911-020-01225-8","target":"_blank"},"children":["Deep learning based feature-level integration of multi-omics data for breast cancer patients survival analysis"]},{"name":"a","attribs":{"href":"https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006076","target":"_blank"},"children":["Cox-nnet: An artificial neural network method for prognosis prediction of high-throughput omics data"]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/8621345","target":"_blank"},"children":["Cox-PASNet: Pathway-based Sparse Deep Neural Network for Survival Analysis"]},{"name":"a","attribs":{"href":"https://www.jmlr.org/papers/volume20/18-424/18-424.pdf?ref=https://githubhelp.com","target":"_blank"},"children":["Time-to-Event Prediction with Neural Networks and Cox Regression"]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-16145-3_9","target":"_blank"},"children":["CRESA: A Deep Learning Approach to Competing Risks, Recurrent Event Survival Analysis"]},{"name":"a","attribs":{"href":"https://www.sciencedirect.com/science/article/pii/S1053811922006218","target":"_blank"},"children":["DAFT: A universal module to interweave tabular data and 3D images in CNNs"]},{"name":"a","attribs":{"href":"https://www.sciencedirect.com/science/article/abs/pii/S0957417418304949","target":"_blank"},"children":["A Deep Active Survival Analysis approach for precision treatment recommendations: Application of prostate cancer"]},{"name":"a","attribs":{"href":"http://proceedings.mlr.press/v80/chapfuwa18a.html","target":"_blank"},"children":["Adversarial Time-to-Event Modeling"]},{"name":"a","attribs":{"href":"https://proceedings.mlr.press/v149/nagpal21a","target":"_blank"},"children":["Deep Cox Mixtures for Survival Regression"]},{"name":"a","attribs":{"href":"https://www.computer.org/csdl/proceedings-article/icdh/2022/814900a169/1G6jNaxaV3y","target":"_blank"},"children":["Deep Learning-Based Discrete Calibrated Survival Prediction"]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/9313333","target":"_blank"},"children":["DeepComp: Which Competing Event Will Hit the Patient First?"]},{"name":"a","attribs":{"href":"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8075516/","target":"_blank"},"children":["DeepCompete : A deep learning approach to competing risks in continuous time domain"]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/7822579","target":"_blank"},"children":["Deep convolutional neural network for survival analysis with pathological images"]},{"name":"a","attribs":{"href":"https://ojs.aaai.org/index.php/AAAI/article/view/11842","target":"_blank"},"children":["DeepHit: A Deep Learning Approach to Survival Analysis with Competing Risks"]},{"name":"a","attribs":{"href":"https://www.sciencedirect.com/science/article/pii/S2001037021001793","target":"_blank"},"children":["DeepOmix: A scalable and interpretable multi-omics deep learning framework and application in cancer survival analysis"]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-031-05936-0_20","target":"_blank"},"children":["DeepPAMM: Deep Piecewise Exponential Additive Mixed Models for Complex Hazard Structures in Survival Analysis"]},{"name":"a","attribs":{"href":"https://bmcmedresmethodol.biomedcentral.com/articles/10.1186/s12874-018-0482-1","target":"_blank"},"children":["DeepSurv: personalized treatment recommender system using a Cox proportional hazards deep neural network"]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-47426-3_53","target":"_blank"},"children":["Estimation of Conditional Mixture Weibull Distribution with Right Censored Data Using Neural Network for Time-to-Event Analysis"]},{"name":"a","attribs":{"href":"https://doi.org/10.1002/sim.8743","target":"_blank"},"children":["Genome-wide association study-based deep learning for survival prediction"]},{"name":"a","attribs":{"href":"https://arxiv.org/abs/1908.02337","target":"_blank"},"children":["DNNSurv: Deep Neural Networks for Survival Analysis Using Pseudo Values"]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-86340-1_15","target":"_blank"},"children":["DPWTE: A Deep Learning Approach to Survival Analysis Using a Parsimonious Mixture of Weibull Distributions"]},{"name":"a","attribs":{"href":"https://ojs.aaai.org/index.php/AAAI/article/view/4407","target":"_blank"},"children":["Deep Recurrent Survival Analysis"]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/9326348","target":"_blank"},"children":["Deep Survival Machines: Fully Parametric Survival Regression and Representation Learning for Censored Data with Competing Risks"]},{"name":"a","attribs":{"href":"https://doi.org/10.1002/sim.8090","target":"_blank"},"children":["Extreme learning machine Cox model for high-dimensionalsurvival analysis"]},{"name":"a","attribs":{"href":"https://www.mdpi.com/2073-4425/10/3/240","target":"_blank"},"children":["Group Lasso Regularized Deep Learning for Cancer Prognosis from Multi-Omics and Clinical Features"]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/8759499","target":"_blank"},"children":["Image-Based Survival Prediction for Lung Cancer Patients Using CNNS"]},{"name":"a","attribs":{"href":"https://proceedings.mlr.press/v146/hu21a.html","target":"_blank"},"children":["Transformer-Based Deep Survival Analysis"]},{"name":"a","attribs":{"href":"https://doi.org/10.1142/9789813235533_0032","target":"_blank"},"children":["Deep Integrative Analysis for Survival Prediction"]},{"name":"a","attribs":{"href":"https://doi.org/10.1002/sim.9310","target":"_blank"},"children":["IDNetwork: A deep Illness-Death Network based on multi-state event history process for versatile disease prognostication"]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-031-09726-3_20","target":"_blank"},"children":["A Multi-constraint Deep Semi-supervised Learning Method for Ovarian Cancer Prognosis Prediction"]},{"name":"a","attribs":{"href":"https://www.nature.com/articles/s41598-021-92799-4","target":"_blank"},"children":["MultiSurv: Long-term cancer survival prediction using multimodal deep learning"]},{"name":"a","attribs":{"href":"https://arxiv.org/abs/1801.05512","target":"_blank"},"children":["Deep Neural Networks for Survival Analysis Based on a Multi-Task Framework"]},{"name":"a","attribs":{"href":"https://www.tandfonline.com/doi/abs/10.1080/00949655.2021.2015770","target":"_blank"},"children":["Deep survival algorithm based on nuclear norm"]},{"name":"a","attribs":{"href":"https://peerj.com/articles/6257/?ref=https%3A%2F%2Fgithubhelp.com","target":"_blank"},"children":["A scalable discrete-time survival model for neural networks"]},{"name":"a","attribs":{"href":"https://link.springer.com/article/10.1007/s10985-021-09532-6","target":"_blank"},"children":["Continuous and Discrete-Time Survival Prediction with Neural Networks"]},{"name":"a","attribs":{"href":"https://www.nature.com/articles/s41467-020-20167-3","target":"_blank"},"children":["A meta-learning approach for genomic survival analysis"]},{"name":"a","attribs":{"href":"https://www.sciencedirect.com/science/article/pii/S0933365718305992","target":"_blank"},"children":["A deep survival analysis method based on ranking"]},{"name":"a","attribs":{"href":"https://europepmc.org/article/med/35801434","target":"_blank"},"children":["Survival prediction model for right-censored data based on improved composite quantile regression neural network"]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-01424-7_3","target":"_blank"},"children":["RNN-SURV: A Deep Recurrent Model for Survival Analysis"]},{"name":"a","attribs":{"href":"https://www.frontiersin.org/articles/10.3389/fgene.2019.00166/full","target":"_blank"},"children":["SALMON: Survival Analysis Learning With Multi-Omics Neural Networks on Breast Cancer"]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/9171987","target":"_blank"},"children":["Survival Analysis For Computing Systems Using A Deep Ensemble Network"]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-87240-3_73","target":"_blank"},"children":["Survival Prediction Based on Histopathology Imaging and Clinical Data: A Novel, Whole Slide CNN Approach"]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/9373895","target":"_blank"},"children":["Deep Semisupervised Multitask Learning Model and Its Interpretability for Survival Analysis"]},{"name":"a","attribs":{"href":"http://www.nature.com/articles/s41598-020-58722-z","target":"_blank"},"children":["DeepBTS: Prediction of Recurrence-free Survival of Non-small Cell Lung Cancer Using a Time-binned Deep Neural Network"]},{"name":"a","attribs":{"href":"https://www.mdpi.com/2072-6694/13/13/3106","target":"_blank"},"children":["SurvCNN: A Discrete Time-to-Event Cancer Survival Estimation Framework Using Image Representations of Omics Data"]},{"name":"a","attribs":{"href":"https://www.frontiersin.org/article/10.3389/fonc.2020.588990","target":"_blank"},"children":["SurvNet: A Novel Deep Neural Network for Lung Cancer Survival Analysis With Missing Values"]},{"name":"a","attribs":{"href":"http://arxiv.org/abs/2006.04893","target":"_blank"},"children":["A General Framework for Survival Analysis and Multi-State Modelling"]},{"name":"a","attribs":{"href":"https://doi.org/10.1145/3535508.3545521","target":"_blank"},"children":["SurvTRACE: Transformers for Survival Analysis with Competing Events"]},{"name":"a","attribs":{"href":"https://www.nature.com/articles/s41746-022-00679-6","target":"_blank"},"children":["Discrete-time survival analysis in the critically ill: a deep learning approach using heterogeneous data"]},{"name":"a","attribs":{"href":"https://doi.org/10.1002/sim.9392","target":"_blank"},"children":["Deep learning for the dynamic prediction of multivariate longitudinal and survival data"]},{"name":"a","attribs":{"href":"https://doi.org/10.1093/bioinformatics/btaa462","target":"_blank"},"children":["Improved survival analysis by learning shared genomic information from pan-cancer data"]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-43823-4_37","target":"_blank"},"children":["A Wide and Deep Neural Network for Survival Analysis from Anatomical Shape and Tabular Clinical Data"]},{"name":"a","attribs":{"href":"https://link.springer.com/article/10.1007/s00180-021-01086-3","target":"_blank"},"children":["Mixture cure rate models with neural network estimated nonparametric components"]}],"sticky":"left","headerStyle":{"position":"sticky","left":0,"background":"#fff","zIndex":1}},{"id":"authors","name":"authors","type":"character","show":false},{"id":"year","name":"year","type":"numeric","show":false,"width":49},{"id":"model name","name":"model name","type":"character"},{"id":"model class","name":"model class","type":"factor","cell":["other","Cox-based","Cox-based","discrete-time","Cox-based","Cox-based","Cox-based","Cox-based","Cox-based","discrete-time","Cox-based","other","other","Cox-based","discrete-time","discrete-time","other","Cox-based","discrete-time","Cox-based","PEM-based","Cox-based","para-metric","Cox-based","discrete-time","para-metric","discrete-time","para-metric","Cox-based","Cox-based","Cox-based","discrete-time","other","PEM-based","Cox-based","discrete-time","discrete-time","Cox-based","discrete-time","PEM-based","Cox-based","other","other","discrete-time","Cox-based","Cox-based","other","discrete-time","discrete-time","discrete-time","Cox-based","other","discrete-time","discrete-time","discrete-time","Cox-based","Cox-based","Cox-based"],"width":60},{"id":"paramet-rization","name":"paramet-rization","type":"character","show":true,"width":70},{"id":"archi-tecture","name":"archi-tecture","type":"factor","width":60},{"id":"multiple losses","name":"multiple losses","type":"character","show":false,"cell":["✔️ Yes","✔️ Yes","❌ no","❌ no","❌ no","✔️ Yes","❌ no","❌ no","❌ no","✔️ Yes","❌ no","n.a.","✔️ Yes","✔️ Yes","✔️ Yes","✔️ Yes","❌ no","❌ no","✔️ Yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ Yes","✔️ Yes","❌ no","❌ no","❌ no","✔️ Yes","❌ no","❌ no","✔️ Yes","❌ no","not described","❌ no","❌ no","❌ no","❌ no","✔️ Yes","❌ no","✔️ Yes","❌ no","❌ no","✔️ Yes","✔️ Yes","❌ no","❌ no","✔️ Yes","no   ","✔️ Yes","❌ no","✔️ Yes","✔️ Yes","❌ no","❌ no"],"width":60},{"id":"left-truncation","name":"left-truncation","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","no  ","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":69},{"id":"interval-censoring","name":"interval-censoring","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","no   ","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":65},{"id":"CR","name":"CR","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","✔️ yes","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":49},{"id":"MSM","name":"MSM","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":49},{"id":"recurrent events","name":"recurrent events","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":64},{"id":"TVF","name":"TVF","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no"],"width":49},{"id":"TVE","name":"TVE","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":49},{"id":"high dimen-sionality","name":"high dimen-sionality","type":"character","cell":["❌ no","❌ no","✔️ yes","✔️ yes","✔️ yes","✔️ yes","✔️ yes","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","✔️ yes","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no"],"width":80},{"id":"multi-modality","name":"multi-modality","type":"character","cell":["❌ no","✔️ yes","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","✔️ yes"],"width":61},{"id":"interpret-ability","name":"interpret-ability","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","assigning biological meaning to nodes","assigning biological meaning to nodes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","assigning biological meaning to nodes","classical statistical interpretability of tabular data","❌ no","❌ no","LIME","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","risk backpropagation based on SurvivalNet","❌ no","❌ no","❌ no","feature importance","❌ no","❌ no","feature importance; feature effect plots on cumulative incidence curves","❌ no","❌ no","❌ no","variational survNode","interpretation of attention maps","SHAP","❌ no","❌ no","classical statistical interpretability of tabular data","classical statistical interpretability of tabular data"],"width":64},{"id":"code","name":"code","type":"character","cell":[{"name":"a","attribs":{"href":"https://git.sr.ht/~aussetg/nfsurvival","target":"_blank"},"children":["git.sr.ht/~aussetg/nfsurvival"]},"❌ No",{"name":"a","attribs":{"href":"https://github.com/wangwangCCChen/CNN-Cox","target":"_blank"},"children":["GitHub repo"]},"❌ No",{"name":"a","attribs":{"href":"https://github.com/shuanggema/CNTsurv","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/tongli1210/BreastCancerSurvivalIntegration","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/lanagarmire/cox-nnet","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/DataX-JieHao/Cox-PASNet","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/havakv/pycox","target":"_blank"},"children":["GitHub repo"]},"❌ No",{"name":"a","attribs":{"href":"https://github.com/ai-med/DAFT","target":"_blank"},"children":["GitHub repo"]},"❌ No",{"name":"a","attribs":{"href":"https://github.com/paidamoyo/adversarial_time_to_event","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/chiragnagpal/deep_cox_mixtures","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/imsb-uke/dcsurv","target":"_blank"},"children":["GitHub repo"]},"❌ No","❌ No","❌ No",{"name":"a","attribs":{"href":"https://github.com/chl8856/DeepHit","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/CancerProfiling/DeepOmix","target":"_blank"},"children":["GitHub repo"]},"❌ No",{"name":"a","attribs":{"href":"https://github.com/jaredleekatzman/DeepSurv","target":"_blank"},"children":["GitHub repo"]},"❌ No",{"name":"a","attribs":{"href":"https://github.com/yingding99/DNNSurv","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/lilizhaoUM/DNNSurv","target":"_blank"},"children":["GitHub repo"]},"❌ No",{"name":"a","attribs":{"href":"https://github.com/rk2900/drsa","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://autonlab.org/auton-survival","target":"_blank"},"children":["autonlab.org/auton-survival"]},{"name":"a","attribs":{"href":"https://github.com/whcsu/ELMCoxBAR","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/WGLab/GDP","target":"_blank"},"children":["GitHub repo"]},"❌ No","❌ No","❌ No","❌ No","❌ No",{"name":"a","attribs":{"href":"https://github.com/luisvalesilva/multisurv","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://square.github.io/pysurvival","target":"_blank"},"children":["GitHub Pages site"]},"❌ No",{"name":"a","attribs":{"href":"https://github.com/MGensheimer/nnet-survival","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/havakv/pycox","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/gevaertlab/metalearning_survival","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/sysucc-ailab/RankDeepSurv","target":"_blank"},"children":["GitHub repo"]},"❌ No","❌ No",{"name":"a","attribs":{"href":"https://github.com/huangzhii/SALMON","target":"_blank"},"children":["GitHub repo"]},"❌ No","❌ No",{"name":"a","attribs":{"href":"https://github.com/LeyiChi/SSMTL","target":"_blank"},"children":["GitHub repo"]},"❌ No",{"name":"a","attribs":{"href":"https://github.com/TeamSundar/SurvCNN","target":"_blank"},"children":["GitHub repo"]},"❌ No",{"name":"a","attribs":{"href":"https://github.com/stefangroha/SurvNODE","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/RyanWangZf/SurvTRACE","target":"_blank"},"children":["GitHub repo"]},{"name":"a","attribs":{"href":"https://github.com/daplaci/ICUsurvive","target":"_blank"},"children":["GitHub repo"]},"❌ No",{"name":"a","attribs":{"href":"https://github.com/SunkyuKim/VAECox","target":"_blank"},"children":["GitHub repo"]},"❌ No","❌ No"],"width":80},{"id":"datasets (non-private)","name":"datasets (non-private)","type":"character","show":false},{"id":"simulation","name":"simulation","type":"character","show":false,"cell":["✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","✔️ yes","❌ no","✔️ yes","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","✔️ yes","✔️ yes","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes"]}],"filterable":true,"searchable":true,"defaultPageSize":5,"showPageSizeOptions":true,"pageSizeOptions":[5,7,10,58],"highlight":true,"bordered":true,"striped":true,"compact":true,"width":"auto","height":"700px","elementId":"table","dataKey":"0311da03ed795e0bfc96a24a0016d654"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>