<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>index</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

<script src="index_files/libs/core-js-2.5.3/shim.min.js"></script>
<script src="index_files/libs/react-17.0.0/react.min.js"></script>
<script src="index_files/libs/react-17.0.0/react-dom.min.js"></script>
<script src="index_files/libs/reactwidget-1.0.0/react-tools.js"></script>
<script src="index_files/libs/htmlwidgets-1.6.2/htmlwidgets.js"></script>
<link href="index_files/libs/reactable-0.4.4/reactable.css" rel="stylesheet">
<script src="index_files/libs/reactable-binding-0.4.4/reactable.js"></script>


<link rel="stylesheet" href="styles.css">
</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-full">

<main class="content column-page" id="quarto-document-content">



<section id="an-overview-table-of-dl-based-survival-methods" class="level1">
<h1>An Overview Table of DL-based Survival Methods</h1>
<section id="disclaimer" class="level2">
<h2 class="anchored" data-anchor-id="disclaimer">Disclaimer</h2>
<div class="disclaimer-text">
<p>The table on this GitHub Pages site has been created as part of the review paper <a href="https://arxiv.org/abs/2305.14961">Deep Learning For Survival Analysis: A Review</a> using the <strong>R</strong> package <strong>reactable</strong> (Lin, 2023). It is licensed under CC BY.</p>
<p>This table is not guaranteed to be complete and will be continuously extended. To do so, we welcome pull requests by adding to <code>data/raw.csv</code> in the <a href="https://github.com/survival-org/DL4Survival">source repository</a> of this site. New methods should comply with the inclusion and exclusion criteria used in the review paper and contain information for all columns in the table. We will then cross-check and subsequently merge new pull requests. Alternatively, you can also contact us directly to notify us of any new methods.</p>
</div>
</section>
<section id="acronyms-abbreviations-and-explanations-of-the-table-columns" class="level2">
<h2 class="anchored" data-anchor-id="acronyms-abbreviations-and-explanations-of-the-table-columns">Acronyms, abbreviations, and explanations of the table columns</h2>
<div class="disclaimer-text">
<p>All of the columns of the table, as well as the corresponding classifications, are described in detail in the review paper.</p>
<ul>
<li><strong>title</strong>: Title of the article introducing the method.</li>
<li><strong>model name</strong>: Official method name if provided in the respective paper; otherwise, the first three letters of the first author’s last name followed by the year of publication.</li>
<li><strong>model class</strong>: See review paper for classification.</li>
<li><strong>parametrization</strong>: See review paper for classification.</li>
<li><strong>architecture</strong>: Which neural network architecture does the method employ?</li>
<li><strong>left-truncation</strong>: Can the method handle left-truncated survival times?</li>
<li><strong>interval-censoring</strong>: Can the method handle interval-censored survival times?</li>
<li><strong>CR</strong>: Can the method handle competing risks?</li>
<li><strong>MSM</strong>: Can the method handle multi-state modeling?</li>
<li><strong>RE</strong>: Can the method handle recurrent events?</li>
<li><strong>TVF</strong>: Can the method handle time-varying features?</li>
<li><strong>TVE</strong>: Can the method estimate time-varying effects?</li>
<li><strong>high dim</strong>: Is the method specifically designed for high-dimensional (e.g., omics) features?</li>
<li><strong>multimodality</strong>: Can the method handle unstructured features or multiple feature modalities (e.g., images)?</li>
<li><strong>interpretability</strong>: Are the results produced by the methods (in particular, the feature effects) interpretable?
<ul>
<li>BP = backpropagation</li>
<li>CIC = cumulative incidence curve</li>
<li>FI = feature importance</li>
<li>classical = classical statistical interpretability of tabular data</li>
</ul></li>
<li><strong>code</strong>: Is the code for the method and experiments publicly available and, if so, where? (Hyperlink to the main repository; further implementations of the method, if available, are shown by expanding the respective row.)</li>
<li><strong>authors</strong>: Authors of the article introducing the method. (<em>hidden</em>)</li>
<li><strong>year</strong>: Year of publication of the article introducing the method. (<em>hidden</em>)</li>
<li><strong>paper</strong>: URL of the publication of the article (hyperlink in the <strong>title</strong> column). (<em>hidden</em>)</li>
<li><strong>multiple losses</strong>: Does the method use more than one loss for training? (<em>hidden</em>)</li>
<li><strong>datasets</strong>: Which accessible (i.e., public or registered access) datasets is the method evaluated on in the respective paper? (<em>hidden</em>)</li>
<li><strong>minimum dataset size</strong>: Size of the smallest dataset used to evaluate the method. (<em>hidden</em>)</li>
<li><strong>simulation</strong>: Is the method additionally evaluated on synthetic data? (<em>hidden</em>)</li>
<li><strong>comments</strong>: Further comments regarding the method (also shown if expanding the row). (<em>hidden</em>)</li>
</ul>
</div>
</section>
<section id="table" class="level2">
<h2 class="anchored" data-anchor-id="table">Table</h2>
<div class="cell">
<div class="cell-output-display">

<button onclick="Reactable.setHiddenColumns('table', prevColumns => { return prevColumns.length === 0 ? ['authors', 'year', 'paper', 'multiple losses', 'datasets', 'simulation', 'comments'] : [] })">Show/hide more columns</button>
<button onclick="Reactable.downloadDataCSV('table', 'table.csv', {sep: ';'})">
<svg aria-hidden="true" role="img" viewbox="0 0 512 512" style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:currentColor;overflow:visible;position:relative;"><path d="M288 32c0-17.7-14.3-32-32-32s-32 14.3-32 32V274.7l-73.4-73.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3l128 128c12.5 12.5 32.8 12.5 45.3 0l128-128c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L288 274.7V32zM64 352c-35.3 0-64 28.7-64 64v32c0 35.3 28.7 64 64 64H448c35.3 0 64-28.7 64-64V416c0-35.3-28.7-64-64-64H346.5l-45.3 45.3c-25 25-65.5 25-90.5 0L165.5 352H64zm368 56a24 24 0 1 1 0 48 24 24 0 1 1 0-48z"></path></svg>
Download as CSV
</button>
<div id="table" class="reactable html-widget " style="width:auto;height:700px;"></div>
<script type="application/json" data-for="table">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"title":["Individual Survival Curves with Conditional Normalizing Flows","Countdown Regression: Sharp and Calibrated Survival Predictions","CapSurv: Capsule Network for Survival Analysis With Whole Slide Pathological Images","A convolutional neural network model for survival prediction based on prognosis-related cascaded Wx feature selection","CNN-based survival model for pancreatic ductal adenocarcinoma in medical imaging","Survival Analysis with High-Dimensional Omics Data Using a Threshold Gradient Descent Regularization-Based Neural Network Approach","Deep learning based feature-level integration of multi-omics data for breast cancer patients survival analysis","Cox-nnet: An artificial neural network method for prognosis prediction of high-throughput omics data","Cox-PASNet: Pathway-based Sparse Deep Neural Network for Survival Analysis","Time-to-Event Prediction with Neural Networks and Cox Regression","CRESA: A Deep Learning Approach to Competing Risks, Recurrent Event Survival Analysis","DAFT: A universal module to interweave tabular data and 3D images in CNNs","A Deep Active Survival Analysis approach for precision treatment recommendations: Application of prostate cancer","Adversarial Time-to-Event Modeling","Deep Cox Mixtures for Survival Regression","Deep Learning-Based Discrete Calibrated Survival Prediction","DeepComp: Which Competing Event Will Hit the Patient First?","DeepCompete : A deep learning approach to competing risks in continuous time domain","Deep convolutional neural network for survival analysis with pathological images","DeepHit: A Deep Learning Approach to Survival Analysis with Competing Risks","DeepOmix: A scalable and interpretable multi-omics deep learning framework and application in cancer survival analysis","DeepPAMM: Deep Piecewise Exponential Additive Mixed Models for Complex Hazard Structures in Survival Analysis","DeepSurv: personalized treatment recommender system using a Cox proportional hazards deep neural network","Estimation of Conditional Mixture Weibull Distribution with Right Censored Data Using Neural Network for Time-to-Event Analysis","Genome-wide association study-based deep learning for survival prediction","DNNSurv: Deep Neural Networks for Survival Analysis Using Pseudo Values","DPWTE: A Deep Learning Approach to Survival Analysis Using a Parsimonious Mixture of Weibull Distributions","Deep Recurrent Survival Analysis","Deep Survival Machines: Fully Parametric Survival Regression and Representation Learning for Censored Data with Competing Risks","Extreme learning machine Cox model for high-dimensionalsurvival analysis","Group Lasso Regularized Deep Learning for Cancer Prognosis from Multi-Omics and Clinical Features","Image-Based Survival Prediction for Lung Cancer Patients Using CNNS","Transformer-Based Deep Survival Analysis","Deep Integrative Analysis for Survival Prediction","IDNetwork: A deep Illness-Death Network based on multi-state event history process for versatile disease prognostication","Estimating Calibrated Individualized Survival Curves with Deep Learning","A Multi-constraint Deep Semi-supervised Learning Method for Ovarian Cancer Prognosis Prediction","MultiSurv: Long-term cancer survival prediction using multimodal deep learning","Deep Neural Networks for Survival Analysis Based on a Multi-Task Framework","Deep survival algorithm based on nuclear norm","A scalable discrete-time survival model for neural networks","PAGE-Net: Interpretable and Integrative Deep Learning for Survival Analysis Using Histopathological Images and Genomic Data","Continuous and Discrete-Time Survival Prediction with Neural Networks","A meta-learning approach for genomic survival analysis","A deep survival analysis method based on ranking","Survival prediction model for right-censored data based on improved composite quantile regression neural network","RNN-SURV: A Deep Recurrent Model for Survival Analysis","SALMON: Survival Analysis Learning With Multi-Omics Neural Networks on Breast Cancer","Survival Analysis For Computing Systems Using A Deep Ensemble Network","Survival Prediction Based on Histopathology Imaging and Clinical Data: A Novel, Whole Slide CNN Approach","Deep Semisupervised Multitask Learning Model and Its Interpretability for Survival Analysis","DeepBTS: Prediction of Recurrence-free Survival of Non-small Cell Lung Cancer Using a Time-binned Deep Neural Network","SurvCNN: A Discrete Time-to-Event Cancer Survival Estimation Framework Using Image Representations of Omics Data","SurvNet: A Novel Deep Neural Network for Lung Cancer Survival Analysis With Missing Values","A General Framework for Survival Analysis and Multi-State Modelling","SurvTRACE: Transformers for Survival Analysis with Competing Events","Discrete-time survival analysis in the critically ill: a deep learning approach using heterogeneous data","Deep learning for the dynamic prediction of multivariate longitudinal and survival data","Improved survival analysis by learning shared genomic information from pan-cancer data","A Wide and Deep Neural Network for Survival Analysis from Anatomical Shape and Tabular Clinical Data","Mixture cure rate models with neural network estimated nonparametric components"],"model name":["Aus2021","Ava2020","CapSurv","CNN-Cox","CNN-Survival","CNT","ConcatAE / CrossAE","Cox-nnet","Cox-PASNet","Cox-Time","CRESA","DAFT","DASA","DATE","DCM","DCS","DeepComp","DeepCompete","DeepConvSurv","DeepHit","DeepOmix","DeepPAMM","DeepSurv","DeepWeiSurv","DNNSurv_Sun2020","DNNSurv_Zha2019","DPWTE","DRSA","DSM","ELMCoxBAR","GDP","Haa2019","Hu2021","Hua2018","IDNetwork","Kam2021","MCAP","MultiSurv","N-MTLR","NN-DeepSurv","Nnet-survival","PAGE-Net","PC-Hazard","Qiu2020","RankDeepSurv","rcICQRNN","RNN-SURV","SALMON","San2020","SSCNN","SSMTL","su-DeepBTS","SurvCNN","SurvNet","survNode","SurvTRACE","Tho2022","TransformerJM","VAECox","WideAndDeep","Xie2021"],"model class":["other","parametric","Cox-based","Cox-based","discrete-time","Cox-based","Cox-based","Cox-based","Cox-based","Cox-based","discrete-time","Cox-based","other","other","Cox-based","discrete-time","discrete-time",null,"Cox-based","discrete-time","Cox-based","PEM-based","Cox-based","parametric","Cox-based","discrete-time","parametric","discrete-time","parametric","Cox-based","Cox-based","Cox-based","discrete-time","other","PEM-based","discrete-time","Cox-based","discrete-time","discrete-time","Cox-based","discrete-time","Cox-based","PEM-based","Cox-based",null,"other","discrete-time","Cox-based","Cox-based",null,"discrete-time","discrete-time","discrete-time","Cox-based",null,"discrete-time","discrete-time","discrete-time","Cox-based","Cox-based","Cox-based"],"parametrization":["parameters of transformation model","parameters of log-normal distribution","hazard rate","hazard rate","discrete hazard","hazard rate","hazard rate","hazard rate","hazard rate","hazard rate","discrete hazard","hazard rate","n.a.","custom loss","hazard rate","survival probability","discrete hazard","hazard rate","hazard rate","PMF","hazard rate","hazard rate","hazard rate","(mixture) Weibull Loss","hazard rate","","(mixture) Weibull Loss","discrete hazard","(mixture) Weibull or Log-Normal loss","hazard rate","hazard rate","hazard rate","discrete hazard","person-specific risk","","survival probability","hazard rate","discrete hazard","parameters of logistic regression","hazard rate","discrete hazard","hazard rate","hazard rate","hazard rate","custom loss","parameters of quantile regression","survival probability","hazard rate","hazard rate","custom loss","custom loss","discrete hazard","discrete hazard","hazard rate","hazard rate","discrete hazard","discrete hazard","PMF","hazard rate","hazard rate","hazard rate"],"architecture":["normalizing flows","flexible","CapsNet","CNN","CNN","FFNN","AE","FFNN","FFNN","FFNN","RNN","CNN + DAFT","SAE","GAN","VAE","RNN","RNN","nODE","CNN","FFNN / RNN","FFNN","flexible","FFNN","FFNN","FFNN","FFNN","FFNN + SWM layers","RNN","FFNN / RNN","ELM","FFNN","CNN  ","transformer","CNN + correlational layer","FFNN","RNN","FFNN","flexible","FFNN","FFNN","flexible","CNN","FFNN","FFNN","FFNN ","FFNN","RNN","FFNN + eigengene modules","FFNN ensemble","Siamese CNN","FFNN","FFNN","CNN","FFNN + context gating","nODE","transformer","RNN","transformer","VAE + transfer learning","PointNet","CNN"],"left-truncation":["no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no  ","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no"],"interval-censoring":["no","yes","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no   ","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no"],"CR":["no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","yes","yes","no","yes","no","yes","no","no","no","no","no","no","yes","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","yes","no","no","no","no","no","no"],"MSM":["no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no"],"RE":["no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no"],"TVF":["no","yes","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","yes","no","yes","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","yes","no","no","no"],"TVE":["no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","no","no","no","yes","no","no","no","no","no","no","no","no","yes","no","no","yes","no","no","no","no","no","yes","no","no","no","yes","no","no","no","no","no","no","no","no","no","no"],"high dim":["no","no","no","yes","yes","yes","yes","yes","yes","no","no","no","no","no","no","no","no","no","no","no","yes","no","no","no","yes","no","no","no","no","no","yes","yes","no","no","no","no","yes","yes","no","no","no","yes","no","yes","no","yes","no","yes","no","no","no","no","yes","no","no","no","no","no","yes","no","no"],"multi-modality":["no","no","yes","no","yes","no","yes","no","no","no","no","yes","no","no","no","no","no","no","yes","no","no","yes","no","no","no","no","no","no","no","no","no","yes","no","yes","no","no","no","yes","no","no","yes","yes","no","no","no","no","no","no","no","yes","no","no","no","no","yes","no","yes","no","no","yes","yes"],"interpret-ability":["no","no","no","no","no","no","no","biological interpretation of architecture","biological interpretation of architecture","no","no","no","no","no","no","no","no","no","no","no","biological interpretation of architecture","classical","no","no","LIME","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","no","biological interpretation of architecture","no","risk BP based on SurvivalNet","no","no","no","FI","no","no","FI; feature effect plots on CIC","no","no","no","variational survNode","attention maps","SHAP","no","no","classical","classical"],"code":["https://git.sr.ht/~aussetg/nfsurvival","http://github.com/stanfordmlgroup/cdr-mimic","no","https://github.com/wangwangCCChen/CNN-Cox","no","https://github.com/shuanggema/CNTsurv","https://github.com/tongli1210/BreastCancerSurvivalIntegration","https://github.com/lanagarmire/cox-nnet","https://github.com/DataX-JieHao/Cox-PASNet","https://github.com/havakv/pycox, https://github.com/RaphaelS1/survivalmodels","no","https://github.com/ai-med/DAFT","no","https://github.com/paidamoyo/adversarial_time_to_event","https://github.com/chiragnagpal/deep_cox_mixtures, https://autonlab.org/auton-survival","https://github.com/imsb-uke/dcsurv","no","no","no","https://github.com/chl8856/DeepHit, https://github.com/havakv/pycox, https://github.com/RaphaelS1/survivalmodels\n","https://github.com/CancerProfiling/DeepOmix","no","https://github.com/jaredleekatzman/DeepSurv, https://github.com/havakv/pycox, https://github.com/RaphaelS1/survivalmodels, https://autonlab.org/auton-survival, https://square.github.io/pysurvival","no","https://github.com/yingding99/DNNSurv","https://github.com/lilizhaoUM/DNNSurv, https://github.com/RaphaelS1/survivalmodels","no","https://github.com/rk2900/drsa","https://autonlab.org/auton-survival","https://github.com/whcsu/ELMCoxBAR","https://github.com/WGLab/GDP","no","no","no","no","https://github.com/MLD3/Calibrated-Survival-Analysis","no","https://github.com/luisvalesilva/multisurv","https://square.github.io/pysurvival","no","https://github.com/MGensheimer/nnet-survival, https://github.com/havakv/pycox, https://github.com/RaphaelS1/survivalmodels","https://github.com/DataX-JieHao/PAGE-Net","https://github.com/havakv/pycox, https://github.com/RaphaelS1/survivalmodels","https://github.com/gevaertlab/metalearning_survival","https://github.com/sysucc-ailab/RankDeepSurv","no","no","https://github.com/huangzhii/SALMON","no","no","https://github.com/LeyiChi/SSMTL","no","https://github.com/TeamSundar/SurvCNN","no","https://github.com/stefangroha/SurvNODE","https://github.com/RyanWangZf/SurvTRACE","https://github.com/daplaci/ICUsurvive","no","https://github.com/SunkyuKim/VAECox","no","no"],"min dataset size":["1638 (WHAS)","51015 (MIMIC III)","98 (LUSC)","10535","520","1093 (TCGA)","1060","5031","522 (GBM)","1904 (METABRIC)","1301 (MIMIC III)","2749 (combined)","~6000","7894 (FLCHAIN)","6524 (FLCHAIN)","1904 (METABRIC)","19830 (AF dataset)","86 (Lung Cancer Dataset)","404","1981 (METABRIC)","241 (KIRC)","614 (BREAST)","1638 (WHAS)","1981 (METABRIC)","7803","5380 (CHS)","1981 (METABRIC)","6036 (CLINIC)","1904 (METABRIC)","","270 (LAML)","232","1904 (METABRIC)","106 (LUSC)","1122 (NCT00275210)","2402 (NACD)","141 (GSE63885)","11315","137 (veteran)","228 (MLC)","9105","447 (combined)","1904 (METABRIC)","9707","1638 (WHAS)","144 (NKI70)","2843 (AIDS2)","583","21068","389","28927 (Lung)","1320","446","1137 (NSCLC)","1904 (METABRIC)","1904 (METABRIC)","29417","1068","9127 (combined for pre-training)","397","352"],"authors":["G. Ausset et al.","Avati et al.","B. Tang et al.","Q. Yin et al.","Y. Zhang et al.","Y. Fanet al.","L. Tong et al.","T. Ching et al.","J. Hao et al.","H. Kvamme et al.","G. Gupta et al.","T. Wolf et al.","M. Nezhad et al.","P. Chapfuwa et al.","C. Nagpal et al.","P. Fuhlert et al.","Y. Li et al.","Aastha et al.","X. Zhu et al.","C. Lee et al.","L. Zhao et al.","P. Kopper et al.","J. Katzman et al.","A. Bennis et al.","T. Sun","L. Zhao et al.","A. Bennis et al.","K. Ren et al.","C. Nagpal et al.","H. Wang et al.","G. Xie et al.","C. Haarburger et al.","S. Hu et al.","C. Huang et al.","A. Cottin et al.","F. Kamran and J. Wiens","H. Chai et al.","L. Vale-Silva and K. Rohr","S. Fotso","J. Tong and X. Zhao","M. Gensheimer and B. Narasimhan","Hao et al.","H. Kvamme and Ø. Borgan ","Y. Qiu et al.","B. Jing et al.","X. Qin et al.","E. Giunchiglia et al.","Z. Huang et al.","B. Sansaengtham et al.","S. Agarwal et al.","S. Chi et al.","B. Lee et al.","Y. Kalakoti et al.","J. Wang et al.","S. Groha et al.","Z. Wang","H. Thorsen-Meyer et al.","J. Lin and S. Luo","S. Kim et al.","S. Pölsterl et al.","Y. Xie and Z. Yu"],"year":[2021,2020,2019,2022,2020,2022,2020,2018,2018,2019,2019,2022,2019,2018,2021,2022,2020,2021,2016,2018,2021,2022,2018,2017,2020,2019,2021,2019,2021,2019,2019,2019,2021,2018,2022,2021,2022,2021,2018,2022,2019,2020,2021,2020,2019,2022,2018,2019,2020,2021,2021,2020,2021,2021,2021,2021,2022,2022,2020,2020,2021],"paper":["https://ieeexplore.ieee.org/document/9564222","http://proceedings.mlr.press/v115/avati20a.html","https://ieeexplore.ieee.org/document/8651474","https://www.nature.com/articles/s41374-022-00801-y","https://bmcmedimaging.biomedcentral.com/articles/10.1186/s12880-020-0418-1","https://www.mdpi.com/2073-4425/13/9/1674","https://bmcmedinformdecismak.biomedcentral.com/articles/10.1186/s12911-020-01225-8","https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006076","https://ieeexplore.ieee.org/document/8621345","https://www.jmlr.org/papers/volume20/18-424/18-424.pdf?ref=https://githubhelp.com","https://link.springer.com/chapter/10.1007/978-3-030-16145-3_9","https://www.sciencedirect.com/science/article/pii/S1053811922006218","https://www.sciencedirect.com/science/article/abs/pii/S0957417418304949","http://proceedings.mlr.press/v80/chapfuwa18a.html","https://proceedings.mlr.press/v149/nagpal21a","https://www.computer.org/csdl/proceedings-article/icdh/2022/814900a169/1G6jNaxaV3y","https://ieeexplore.ieee.org/document/9313333","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8075516/","https://ieeexplore.ieee.org/document/7822579","https://ojs.aaai.org/index.php/AAAI/article/view/11842","https://www.sciencedirect.com/science/article/pii/S2001037021001793","https://link.springer.com/chapter/10.1007/978-3-031-05936-0_20","https://bmcmedresmethodol.biomedcentral.com/articles/10.1186/s12874-018-0482-1","https://link.springer.com/chapter/10.1007/978-3-030-47426-3_53","https://doi.org/10.1002/sim.8743","https://arxiv.org/abs/1908.02337","https://link.springer.com/chapter/10.1007/978-3-030-86340-1_15","https://ojs.aaai.org/index.php/AAAI/article/view/4407","https://ieeexplore.ieee.org/document/9326348","https://doi.org/10.1002/sim.8090","https://www.mdpi.com/2073-4425/10/3/240","https://ieeexplore.ieee.org/document/8759499","https://proceedings.mlr.press/v146/hu21a.html","https://doi.org/10.1142/9789813235533_0032","https://doi.org/10.1002/sim.9310","https://doi.org/10.1609/aaai.v35i1.16098","https://link.springer.com/chapter/10.1007/978-3-031-09726-3_20","https://www.nature.com/articles/s41598-021-92799-4","https://arxiv.org/abs/1801.05512","https://www.tandfonline.com/doi/abs/10.1080/00949655.2021.2015770","https://peerj.com/articles/6257/?ref=https%3A%2F%2Fgithubhelp.com","https://doi.org/10.1142/9789811215636_0032","https://link.springer.com/article/10.1007/s10985-021-09532-6","https://www.nature.com/articles/s41467-020-20167-3","https://www.sciencedirect.com/science/article/pii/S0933365718305992","https://europepmc.org/article/med/35801434","https://link.springer.com/chapter/10.1007/978-3-030-01424-7_3","https://www.frontiersin.org/articles/10.3389/fgene.2019.00166/full","https://ieeexplore.ieee.org/document/9171987","https://link.springer.com/chapter/10.1007/978-3-030-87240-3_73","https://ieeexplore.ieee.org/document/9373895","http://www.nature.com/articles/s41598-020-58722-z","https://www.mdpi.com/2072-6694/13/13/3106","https://www.frontiersin.org/article/10.3389/fonc.2020.588990","http://arxiv.org/abs/2006.04893","https://doi.org/10.1145/3535508.3545521","https://www.nature.com/articles/s41746-022-00679-6","https://doi.org/10.1002/sim.9392","https://doi.org/10.1093/bioinformatics/btaa462","https://link.springer.com/chapter/10.1007/978-3-030-43823-4_37","https://link.springer.com/article/10.1007/s00180-021-01086-3"],"multiple losses":["yes","no","yes","no","no","no","yes","no","no","no","yes","no","n.a.","yes","yes","yes","yes","no","no","yes","no","no","no","no","no","no","no","yes","yes","no","no","no","yes","no","no","yes","yes","no","not described","no","no","no","no","no","yes","no","yes","no","no","yes","yes","no","no","yes","no   ","yes","no","yes","yes","no","no"],"datasets":["METABRIC, RGBSG, SUPPORT, WHAS","MIMIC III, STARR","TCGA (GBM, LUSC)","TCGA (pan-cancer datasets)","Lung1 (TCIA)","HGSOC, TCGA","TCGA (BRCA)","TCGA (pan-cancer datasets)","TCGA (GBM, OV)","FLCHAIN, KKBox churn, METABRIC, RGBSG, SUPPORT","MIMIC III, Engines Failure dataset (proprietary)","ADNI, AIBL","SEER-Medicare prostate cancer dataset","EHR, FLCHAIN, SEER, SUPPORT","FLCHAIN, SEER, SUPPORT","FLCHAIN, METABRIC, SUPPORT","AF dataset, SEER","AML, DBCD, DLBCL, Lung Cancer Dataset, NSBCD, SEER breast cancer dataset","NLST","METABRIC, SEER, UNOS","TCGA (BLCA, BRCA, HNSC, KIRC, LGG, LUAD, OV, STAD)\n","BREAST, GBSG2, ICU, METABRIC, MGUS2, TUMOR","METABRIC, RGBSG, SUPPORT, WHAS","METABRIC, SEER breast cancer dataset","AREDS","CHS, MESA","METABRIC, SEER, SUPPORT","BIDDING, CLINIC, MUSIC","METABRIC, SEER, SUPPORT","DrAsGiven, EMTAB386, GSE14764, GSE32062, GSE49997, LungBeer, METABRIC, NSBCD, TCGAmirna","TCGA (BLCA, ESCA, GBM, HNSC, KIRC, LAML, LIHC, LUAD, LUSC, OV, PAAD, SARC, SKCM, STAD)","Lung1 (TCIA)","METABRIC, SUPPORT","TCGA (GBM, LUSC)","METABRIC, NCT00079274, NCT00275210","CLINIC, NACD","GSE17260, GSE26193, GSE26712, GSE32062, GSE53963, GSE63885, TCGA-OV","TCGA","veteran, WHAS","METABRIC, MIMIC III, MLC, RGBSG, SUPPORT, WHAS","SUPPORT","TCGA, TCIA (GBM gene expression data)","FLCHAIN, METABRIC, NWTCO, RGBSG, SUPPORT","TCGA (pan-cancer datasets)","METABRIC, RGBSG, SUPPORT, WHAS","METABRIC, NKI70, TCC","AIDS2, FLCHAIN, NWTCO, UNOS Transplant, UNOS Waitlist","Broad GDAC Firehose (BRCA)","Google Cloud Cluster Dataset","TCGA (GBM)","SEER (CRC, Lung, Breast, Prostate)","Catholic Medical Center (NSCLC)","TCGA","West China Hospital IB-IIA stage NSCLC dataset, SEER","METABRIC, SUPPORT","METABRIC, SUPPORT","Danish Electronic Patient Records (Registered Access)","ADNI, NACC","TCGA (BLCA, BRCA, HNSC, KIRC, LGG, LIHC, LUAD, LUSC, OV, STAD, CESC, COAD, GBM, KIRP, LAML, PAAD, PRAD, SKCM, THCA, UCEC)","ADNI","OASIS-3"],"simulation":["yes","no","no","no","no","yes","no","yes","no","yes","no","no","no","no","no","yes","no","no","no","yes","no","yes","yes","no","yes","yes","no","no","no","yes","yes","no","no","no","yes","no","no","no","yes","yes","yes","no","yes","no","no","yes","no","no","no","no","no","no","no","no","yes","no","no","yes","no","no","yes"],"comments":["","RNN architecture choice for handling TVF (STARR)","","","","","","","","","","","","","","","","","","RNN architecture and TVF modeling via extension \"Dynamic-DeepHit\"","","","","","","","","","RNN architecture and TVF modeling via extension \"RDSM\"","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""]},"columns":[{"id":"title","name":"title","type":"character","show":true,"cell":[{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/9564222","target":"_blank"},"children":["Individual Survival Curves with Conditional Normalizing Flows"]},{"name":"a","attribs":{"href":"http://proceedings.mlr.press/v115/avati20a.html","target":"_blank"},"children":["Countdown Regression: Sharp and Calibrated Survival Predictions"]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/8651474","target":"_blank"},"children":["CapSurv: Capsule Network for Survival Analysis With Whole Slide ..."]},{"name":"a","attribs":{"href":"https://www.nature.com/articles/s41374-022-00801-y","target":"_blank"},"children":["A convolutional neural network model for survival prediction bas..."]},{"name":"a","attribs":{"href":"https://bmcmedimaging.biomedcentral.com/articles/10.1186/s12880-020-0418-1","target":"_blank"},"children":["CNN-based survival model for pancreatic ductal adenocarcinoma in..."]},{"name":"a","attribs":{"href":"https://www.mdpi.com/2073-4425/13/9/1674","target":"_blank"},"children":["Survival Analysis with High-Dimensional Omics Data Using a Thres..."]},{"name":"a","attribs":{"href":"https://bmcmedinformdecismak.biomedcentral.com/articles/10.1186/s12911-020-01225-8","target":"_blank"},"children":["Deep learning based feature-level integration of multi-omics dat..."]},{"name":"a","attribs":{"href":"https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006076","target":"_blank"},"children":["Cox-nnet: An artificial neural network method for prognosis pred..."]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/8621345","target":"_blank"},"children":["Cox-PASNet: Pathway-based Sparse Deep Neural Network for Surviva..."]},{"name":"a","attribs":{"href":"https://www.jmlr.org/papers/volume20/18-424/18-424.pdf?ref=https://githubhelp.com","target":"_blank"},"children":["Time-to-Event Prediction with Neural Networks and Cox Regression"]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-16145-3_9","target":"_blank"},"children":["CRESA: A Deep Learning Approach to Competing Risks, Recurrent Ev..."]},{"name":"a","attribs":{"href":"https://www.sciencedirect.com/science/article/pii/S1053811922006218","target":"_blank"},"children":["DAFT: A universal module to interweave tabular data and 3D image..."]},{"name":"a","attribs":{"href":"https://www.sciencedirect.com/science/article/abs/pii/S0957417418304949","target":"_blank"},"children":["A Deep Active Survival Analysis approach for precision treatment..."]},{"name":"a","attribs":{"href":"http://proceedings.mlr.press/v80/chapfuwa18a.html","target":"_blank"},"children":["Adversarial Time-to-Event Modeling"]},{"name":"a","attribs":{"href":"https://proceedings.mlr.press/v149/nagpal21a","target":"_blank"},"children":["Deep Cox Mixtures for Survival Regression"]},{"name":"a","attribs":{"href":"https://www.computer.org/csdl/proceedings-article/icdh/2022/814900a169/1G6jNaxaV3y","target":"_blank"},"children":["Deep Learning-Based Discrete Calibrated Survival Prediction"]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/9313333","target":"_blank"},"children":["DeepComp: Which Competing Event Will Hit the Patient First?"]},{"name":"a","attribs":{"href":"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8075516/","target":"_blank"},"children":["DeepCompete : A deep learning approach to competing risks in con..."]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/7822579","target":"_blank"},"children":["Deep convolutional neural network for survival analysis with pat..."]},{"name":"a","attribs":{"href":"https://ojs.aaai.org/index.php/AAAI/article/view/11842","target":"_blank"},"children":["DeepHit: A Deep Learning Approach to Survival Analysis with Comp..."]},{"name":"a","attribs":{"href":"https://www.sciencedirect.com/science/article/pii/S2001037021001793","target":"_blank"},"children":["DeepOmix: A scalable and interpretable multi-omics deep learning..."]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-031-05936-0_20","target":"_blank"},"children":["DeepPAMM: Deep Piecewise Exponential Additive Mixed Models for C..."]},{"name":"a","attribs":{"href":"https://bmcmedresmethodol.biomedcentral.com/articles/10.1186/s12874-018-0482-1","target":"_blank"},"children":["DeepSurv: personalized treatment recommender system using a Cox ..."]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-47426-3_53","target":"_blank"},"children":["Estimation of Conditional Mixture Weibull Distribution with Righ..."]},{"name":"a","attribs":{"href":"https://doi.org/10.1002/sim.8743","target":"_blank"},"children":["Genome-wide association study-based deep learning for survival p..."]},{"name":"a","attribs":{"href":"https://arxiv.org/abs/1908.02337","target":"_blank"},"children":["DNNSurv: Deep Neural Networks for Survival Analysis Using Pseudo..."]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-86340-1_15","target":"_blank"},"children":["DPWTE: A Deep Learning Approach to Survival Analysis Using a Par..."]},{"name":"a","attribs":{"href":"https://ojs.aaai.org/index.php/AAAI/article/view/4407","target":"_blank"},"children":["Deep Recurrent Survival Analysis"]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/9326348","target":"_blank"},"children":["Deep Survival Machines: Fully Parametric Survival Regression and..."]},{"name":"a","attribs":{"href":"https://doi.org/10.1002/sim.8090","target":"_blank"},"children":["Extreme learning machine Cox model for high-dimensionalsurvival ..."]},{"name":"a","attribs":{"href":"https://www.mdpi.com/2073-4425/10/3/240","target":"_blank"},"children":["Group Lasso Regularized Deep Learning for Cancer Prognosis from ..."]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/8759499","target":"_blank"},"children":["Image-Based Survival Prediction for Lung Cancer Patients Using CNNS"]},{"name":"a","attribs":{"href":"https://proceedings.mlr.press/v146/hu21a.html","target":"_blank"},"children":["Transformer-Based Deep Survival Analysis"]},{"name":"a","attribs":{"href":"https://doi.org/10.1142/9789813235533_0032","target":"_blank"},"children":["Deep Integrative Analysis for Survival Prediction"]},{"name":"a","attribs":{"href":"https://doi.org/10.1002/sim.9310","target":"_blank"},"children":["IDNetwork: A deep Illness-Death Network based on multi-state eve..."]},{"name":"a","attribs":{"href":"https://doi.org/10.1609/aaai.v35i1.16098","target":"_blank"},"children":["Estimating Calibrated Individualized Survival Curves with Deep L..."]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-031-09726-3_20","target":"_blank"},"children":["A Multi-constraint Deep Semi-supervised Learning Method for Ovar..."]},{"name":"a","attribs":{"href":"https://www.nature.com/articles/s41598-021-92799-4","target":"_blank"},"children":["MultiSurv: Long-term cancer survival prediction using multimodal..."]},{"name":"a","attribs":{"href":"https://arxiv.org/abs/1801.05512","target":"_blank"},"children":["Deep Neural Networks for Survival Analysis Based on a Multi-Task..."]},{"name":"a","attribs":{"href":"https://www.tandfonline.com/doi/abs/10.1080/00949655.2021.2015770","target":"_blank"},"children":["Deep survival algorithm based on nuclear norm"]},{"name":"a","attribs":{"href":"https://peerj.com/articles/6257/?ref=https%3A%2F%2Fgithubhelp.com","target":"_blank"},"children":["A scalable discrete-time survival model for neural networks"]},{"name":"a","attribs":{"href":"https://doi.org/10.1142/9789811215636_0032","target":"_blank"},"children":["PAGE-Net: Interpretable and Integrative Deep Learning for Surviv..."]},{"name":"a","attribs":{"href":"https://link.springer.com/article/10.1007/s10985-021-09532-6","target":"_blank"},"children":["Continuous and Discrete-Time Survival Prediction with Neural Networks"]},{"name":"a","attribs":{"href":"https://www.nature.com/articles/s41467-020-20167-3","target":"_blank"},"children":["A meta-learning approach for genomic survival analysis"]},{"name":"a","attribs":{"href":"https://www.sciencedirect.com/science/article/pii/S0933365718305992","target":"_blank"},"children":["A deep survival analysis method based on ranking"]},{"name":"a","attribs":{"href":"https://europepmc.org/article/med/35801434","target":"_blank"},"children":["Survival prediction model for right-censored data based on impro..."]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-01424-7_3","target":"_blank"},"children":["RNN-SURV: A Deep Recurrent Model for Survival Analysis"]},{"name":"a","attribs":{"href":"https://www.frontiersin.org/articles/10.3389/fgene.2019.00166/full","target":"_blank"},"children":["SALMON: Survival Analysis Learning With Multi-Omics Neural Netwo..."]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/9171987","target":"_blank"},"children":["Survival Analysis For Computing Systems Using A Deep Ensemble Network"]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-87240-3_73","target":"_blank"},"children":["Survival Prediction Based on Histopathology Imaging and Clinical..."]},{"name":"a","attribs":{"href":"https://ieeexplore.ieee.org/document/9373895","target":"_blank"},"children":["Deep Semisupervised Multitask Learning Model and Its Interpretab..."]},{"name":"a","attribs":{"href":"http://www.nature.com/articles/s41598-020-58722-z","target":"_blank"},"children":["DeepBTS: Prediction of Recurrence-free Survival of Non-small Cel..."]},{"name":"a","attribs":{"href":"https://www.mdpi.com/2072-6694/13/13/3106","target":"_blank"},"children":["SurvCNN: A Discrete Time-to-Event Cancer Survival Estimation Fra..."]},{"name":"a","attribs":{"href":"https://www.frontiersin.org/article/10.3389/fonc.2020.588990","target":"_blank"},"children":["SurvNet: A Novel Deep Neural Network for Lung Cancer Survival An..."]},{"name":"a","attribs":{"href":"http://arxiv.org/abs/2006.04893","target":"_blank"},"children":["A General Framework for Survival Analysis and Multi-State Modelling"]},{"name":"a","attribs":{"href":"https://doi.org/10.1145/3535508.3545521","target":"_blank"},"children":["SurvTRACE: Transformers for Survival Analysis with Competing Events"]},{"name":"a","attribs":{"href":"https://www.nature.com/articles/s41746-022-00679-6","target":"_blank"},"children":["Discrete-time survival analysis in the critically ill: a deep le..."]},{"name":"a","attribs":{"href":"https://doi.org/10.1002/sim.9392","target":"_blank"},"children":["Deep learning for the dynamic prediction of multivariate longitu..."]},{"name":"a","attribs":{"href":"https://doi.org/10.1093/bioinformatics/btaa462","target":"_blank"},"children":["Improved survival analysis by learning shared genomic informatio..."]},{"name":"a","attribs":{"href":"https://link.springer.com/chapter/10.1007/978-3-030-43823-4_37","target":"_blank"},"children":["A Wide and Deep Neural Network for Survival Analysis from Anatom..."]},{"name":"a","attribs":{"href":"https://link.springer.com/article/10.1007/s00180-021-01086-3","target":"_blank"},"children":["Mixture cure rate models with neural network estimated nonparame..."]}],"width":100,"sticky":"left","style":{"borderRight":"1px solid #eee"}},{"id":"model name","name":"model name","type":"character","details":[null,"RNN architecture choice for handling TVF (STARR)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"RNN architecture and TVF modeling via extension \"Dynamic-DeepHit\"",null,null,null,null,null,null,null,null,"RNN architecture and TVF modeling via extension \"RDSM\"",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"width":90},{"id":"model class","name":"model class","type":"factor","width":80},{"id":"parametrization","name":"parametrization","type":"character","show":true,"width":100},{"id":"architecture","name":"architecture","type":"character","width":80},{"id":"left-truncation","name":"left-truncation","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","no  ","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":69},{"id":"interval-censoring","name":"interval-censoring","type":"character","cell":["❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","no   ","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":65},{"id":"CR","name":"CR","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","✔️ yes","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":49},{"id":"MSM","name":"MSM","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":49},{"id":"RE","name":"RE","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":49},{"id":"TVF","name":"TVF","type":"character","cell":["❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no"],"width":49},{"id":"TVE","name":"TVE","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no"],"width":49},{"id":"high dim","name":"high dim","type":"character","cell":["❌ no","❌ no","❌ no","✔️ yes","✔️ yes","✔️ yes","✔️ yes","✔️ yes","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","✔️ yes","❌ no","❌ no","❌ no","❌ no","✔️ yes","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no"],"width":49},{"id":"multi-modality","name":"multi-modality","type":"character","cell":["❌ no","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","✔️ yes"],"width":61},{"id":"interpret-ability","name":"interpret-ability","type":"character","cell":["❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","biological interpretation of architecture","biological interpretation of architecture","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","biological interpretation of architecture","classical","❌ no","❌ no","LIME","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","biological interpretation of architecture","❌ no","risk BP based on SurvivalNet","❌ no","❌ no","❌ no","FI","❌ no","❌ no","FI; feature effect plots on CIC","❌ no","❌ no","❌ no","variational survNode","attention maps","SHAP","❌ no","❌ no","classical","classical"],"width":80},{"id":"code","name":"code","type":"character","cell":[{"name":"a","attribs":{"href":"https://git.sr.ht/~aussetg/nfsurvival","target":"_blank"},"children":["git.sr.ht/~aussetg/nfsurvival"]},{"name":"a","attribs":{"href":"http://github.com/stanfordmlgroup/cdr-mimic","target":"_blank"},"children":["http://github.com/stanfordmlgroup/cdr-mimic"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/wangwangCCChen/CNN-Cox","target":"_blank"},"children":["github.com/wangwangCCChen/CNN-Cox"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/shuanggema/CNTsurv","target":"_blank"},"children":["github.com/shuanggema/CNTsurv"]},{"name":"a","attribs":{"href":"https://github.com/tongli1210/BreastCancerSurvivalIntegration","target":"_blank"},"children":["github.com/tongli1210/BreastCancerSurvivalIntegration"]},{"name":"a","attribs":{"href":"https://github.com/lanagarmire/cox-nnet","target":"_blank"},"children":["github.com/lanagarmire/cox-nnet"]},{"name":"a","attribs":{"href":"https://github.com/DataX-JieHao/Cox-PASNet","target":"_blank"},"children":["github.com/DataX-JieHao/Cox-PASNet"]},{"name":"a","attribs":{"href":"https://github.com/havakv/pycox","target":"_blank"},"children":["github.com/havakv/pycox"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/ai-med/DAFT","target":"_blank"},"children":["github.com/ai-med/DAFT"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/paidamoyo/adversarial_time_to_event","target":"_blank"},"children":["github.com/paidamoyo/adversarial_time_to_event"]},{"name":"a","attribs":{"href":"https://github.com/chiragnagpal/deep_cox_mixtures","target":"_blank"},"children":["github.com/chiragnagpal/deep_cox_mixtures"]},{"name":"a","attribs":{"href":"https://github.com/imsb-uke/dcsurv","target":"_blank"},"children":["github.com/imsb-uke/dcsurv"]},"❌ not accessible","❌ not accessible","❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/chl8856/DeepHit","target":"_blank"},"children":["github.com/chl8856/DeepHit"]},{"name":"a","attribs":{"href":"https://github.com/CancerProfiling/DeepOmix","target":"_blank"},"children":["github.com/CancerProfiling/DeepOmix"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/jaredleekatzman/DeepSurv","target":"_blank"},"children":["github.com/jaredleekatzman/DeepSurv"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/yingding99/DNNSurv","target":"_blank"},"children":["github.com/yingding99/DNNSurv"]},{"name":"a","attribs":{"href":"https://github.com/lilizhaoUM/DNNSurv","target":"_blank"},"children":["github.com/lilizhaoUM/DNNSurv"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/rk2900/drsa","target":"_blank"},"children":["github.com/rk2900/drsa"]},{"name":"a","attribs":{"href":"https://autonlab.org/auton-survival","target":"_blank"},"children":["autonlab.org/auton-survival"]},{"name":"a","attribs":{"href":"https://github.com/whcsu/ELMCoxBAR","target":"_blank"},"children":["github.com/whcsu/ELMCoxBAR"]},{"name":"a","attribs":{"href":"https://github.com/WGLab/GDP","target":"_blank"},"children":["github.com/WGLab/GDP"]},"❌ not accessible","❌ not accessible","❌ not accessible","❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/MLD3/Calibrated-Survival-Analysis","target":"_blank"},"children":["github.com/MLD3/Calibrated-Survival-Analysis"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/luisvalesilva/multisurv","target":"_blank"},"children":["github.com/luisvalesilva/multisurv"]},{"name":"a","attribs":{"href":"https://square.github.io/pysurvival","target":"_blank"},"children":["square.github.io/pysurvival"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/MGensheimer/nnet-survival","target":"_blank"},"children":["github.com/MGensheimer/nnet-survival"]},{"name":"a","attribs":{"href":"https://github.com/DataX-JieHao/PAGE-Net","target":"_blank"},"children":["github.com/DataX-JieHao/PAGE-Net"]},{"name":"a","attribs":{"href":"https://github.com/havakv/pycox","target":"_blank"},"children":["github.com/havakv/pycox"]},{"name":"a","attribs":{"href":"https://github.com/gevaertlab/metalearning_survival","target":"_blank"},"children":["github.com/gevaertlab/metalearning_survival"]},{"name":"a","attribs":{"href":"https://github.com/sysucc-ailab/RankDeepSurv","target":"_blank"},"children":["github.com/sysucc-ailab/RankDeepSurv"]},"❌ not accessible","❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/huangzhii/SALMON","target":"_blank"},"children":["github.com/huangzhii/SALMON"]},"❌ not accessible","❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/LeyiChi/SSMTL","target":"_blank"},"children":["github.com/LeyiChi/SSMTL"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/TeamSundar/SurvCNN","target":"_blank"},"children":["github.com/TeamSundar/SurvCNN"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/stefangroha/SurvNODE","target":"_blank"},"children":["github.com/stefangroha/SurvNODE"]},{"name":"a","attribs":{"href":"https://github.com/RyanWangZf/SurvTRACE","target":"_blank"},"children":["github.com/RyanWangZf/SurvTRACE"]},{"name":"a","attribs":{"href":"https://github.com/daplaci/ICUsurvive","target":"_blank"},"children":["github.com/daplaci/ICUsurvive"]},"❌ not accessible",{"name":"a","attribs":{"href":"https://github.com/SunkyuKim/VAECox","target":"_blank"},"children":["github.com/SunkyuKim/VAECox"]},"❌ not accessible","❌ not accessible"],"details":[null,null,null,null,null,null,null,null,null,{"name":"div","attribs":{},"children":["further implementations of ","Cox-Time",": ",{"name":"a","attribs":{"href":"https://github.com/RaphaelS1/survivalmodels","target":"_blank"},"children":["https://github.com/RaphaelS1/survivalmodels"]}]},null,null,null,null,{"name":"div","attribs":{},"children":["further implementations of ","DCM",": ",{"name":"a","attribs":{"href":"https://autonlab.org/auton-survival","target":"_blank"},"children":["https://autonlab.org/auton-survival"]}]},null,null,null,null,{"name":"div","attribs":{},"children":["further implementations of ","DeepHit",": ",{"name":"a","attribs":{"href":"https://github.com/havakv/pycox","target":"_blank"},"children":["https://github.com/havakv/pycox"]},", ",{"name":"a","attribs":{"href":"https://github.com/RaphaelS1/survivalmodels\n","target":"_blank"},"children":["https://github.com/RaphaelS1/survivalmodels\n"]}]},null,null,{"name":"div","attribs":{},"children":["further implementations of ","DeepSurv",": ",{"name":"a","attribs":{"href":"https://github.com/havakv/pycox","target":"_blank"},"children":["https://github.com/havakv/pycox"]},", ",{"name":"a","attribs":{"href":"https://github.com/RaphaelS1/survivalmodels","target":"_blank"},"children":["https://github.com/RaphaelS1/survivalmodels"]},", ",{"name":"a","attribs":{"href":"https://autonlab.org/auton-survival","target":"_blank"},"children":["https://autonlab.org/auton-survival"]},", ",{"name":"a","attribs":{"href":"https://square.github.io/pysurvival","target":"_blank"},"children":["https://square.github.io/pysurvival"]}]},null,null,{"name":"div","attribs":{},"children":["further implementations of ","DNNSurv_Zha2019",": ",{"name":"a","attribs":{"href":"https://github.com/RaphaelS1/survivalmodels","target":"_blank"},"children":["https://github.com/RaphaelS1/survivalmodels"]}]},null,null,null,null,null,null,null,null,null,null,null,null,null,null,{"name":"div","attribs":{},"children":["further implementations of ","Nnet-survival",": ",{"name":"a","attribs":{"href":"https://github.com/havakv/pycox","target":"_blank"},"children":["https://github.com/havakv/pycox"]},", ",{"name":"a","attribs":{"href":"https://github.com/RaphaelS1/survivalmodels","target":"_blank"},"children":["https://github.com/RaphaelS1/survivalmodels"]}]},null,{"name":"div","attribs":{},"children":["further implementations of ","PC-Hazard",": ",{"name":"a","attribs":{"href":"https://github.com/RaphaelS1/survivalmodels","target":"_blank"},"children":["https://github.com/RaphaelS1/survivalmodels"]}]},null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"width":80},{"id":"min dataset size","name":"min dataset size","type":"character","show":false},{"id":"authors","name":"authors","type":"character","show":false},{"id":"year","name":"year","type":"numeric","show":false,"width":49},{"id":"paper","name":"paper","type":"character","show":false},{"id":"multiple losses","name":"multiple losses","type":"character","show":false,"cell":["✔️ Yes","❌ no","✔️ Yes","❌ no","❌ no","❌ no","✔️ Yes","❌ no","❌ no","❌ no","✔️ Yes","❌ no","n.a.","✔️ Yes","✔️ Yes","✔️ Yes","✔️ Yes","❌ no","❌ no","✔️ Yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ Yes","✔️ Yes","❌ no","❌ no","❌ no","✔️ Yes","❌ no","❌ no","✔️ Yes","✔️ Yes","❌ no","not described","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ Yes","❌ no","✔️ Yes","❌ no","❌ no","✔️ Yes","✔️ Yes","❌ no","❌ no","✔️ Yes","no   ","✔️ Yes","❌ no","✔️ Yes","✔️ Yes","❌ no","❌ no"],"width":60},{"id":"datasets","name":"datasets","type":"character","show":false},{"id":"simulation","name":"simulation","type":"character","show":false,"cell":["✔️ yes","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","✔️ yes","✔️ yes","❌ no","✔️ yes","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","✔️ yes","✔️ yes","✔️ yes","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes","❌ no","❌ no","✔️ yes"]},{"id":"comments","name":"comments","type":"character","show":false}],"filterable":true,"searchable":true,"defaultSorted":[{"id":"model name","desc":false}],"defaultPageSize":7,"showPageSizeOptions":true,"pageSizeOptions":[5,7,10,61],"highlight":true,"bordered":true,"striped":true,"compact":true,"width":"auto","height":"700px","elementId":"table","dataKey":"b219025c07e9debdd07807c495500d2f"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>